lib/generators/rspec/generator/generator_generator.rb

```diff
@@ -0,0 +1,26 @@
+require 'generators/rspec'
+
+if ::Rails::VERSION::STRING >= '5.1'
```

Comment:
Why only Rails 5.1 and above? All Rails versions have generators?

Reply:
Thanks for pointing this out!  I was looking at the system generator to help with creating this generator and forgot to take that out.  But it is now removed in the new commit.